.flaskenv

```diff
@@ -1,2 +1 @@
 FLASK_APP=backend.api:create_app()
-POSTGRES_DB=police_data
```

Comment:
My opinion is `.flaskenv` is a pretty bad spot for environment variable names except for `FLASK_APP=?`. The reason why it's not a good spot for stuff is because there should be one source of truth for the environment and the `config.py` is a better spot. I made a mistake by initially putting `POSTGRES_DB` in here, so I'm deleting.